proto/kuksa/val/v2/val.proto

```diff
@@ -280,7 +282,12 @@ message BatchActuateStreamRequest {
   repeated ActuateRequest actuate_requests = 1;
 }
 
+// Message that can be used by provider to indicate that an actuation request was not accepted.
+// It is optional for the provider to use this message, and for performance reasons it is recommended
```

Comment:
I don't really disagree with what you said here, just to qualify it a bit, I think it won't add a big overhead if the provider returns an OK response in case of success, since kuksa_client actuate is an unary call (it is not supposed to be used with a high frequency as publish datapoint stream). If we add one day actuate_stream to kuksa_client, it should not definitely return an OK response up success to reduce stream overhead.

Reply:
I have no problems changing it so that it is expected that a result is returned.